flare/ase/otf.py

```diff
@@ -260,8 +272,26 @@ def update_gp(self, train_atoms, dft_frcs, dft_energy=None, dft_stress=None):
             flare_stress = None
 
         # update gp model
+        try:
```

Comment:
A comment justifying this try/except block would be helpful (e.g. why is a deepcopy needed here?)

Reply:
added in commit f3c0b55